flow/project.py

```diff
@@ -3153,17 +3200,17 @@ def _job_operations(self, job, ignore_conditions=IgnoreConditions.NONE):
         "Yield instances of JobOperation constructed for specific jobs."
         for name in self.operations:
             group = self._groups[name]
-            yield from group._create_run_job_operations(entrypoint=self._entrypoint, job=job,
+            yield from group._create_run_job_operations(entrypoint=self._entrypoint, jobs=[job],
                                                         default_directives=dict(),
                                                         ignore_conditions=ignore_conditions,
                                                         index=0)
 
     def next_operations(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
-        """Determine the next eligible operations for jobs.
+        r"""Determine the next eligible operations for jobs.
 
-        :param jobs:
+        :param \*jobs:
             The signac job handles.
-        :type job:
+        :type \*jobs:
```

Comment:
This still has to have the same behavior as before since it is user facing. The changes will exist though in private method that takes over this functionality #325.